org-code-javabuilder/theater/src/main/java/org/code/Pixel.java

```diff
@@ -0,0 +1,57 @@
+package org.code;
+
+public class Pixel {
```

Comment:
this could always be added later, but should there be a getter/setter for red, green, and blue? (e.g., `getColor()` and `setColor(int r, int g, int b)`)

kind of tangential, but this question then makes me wonder if there should be a `Color` type that can express colors as an int[] or string. then, (excuse my weird ruby/java pseudocode...) `Image#clear(String color)` would become `Image#clear(Color color)`. this does add some complexity to the student interaction, though, as they'd have to understand the multiple ways a color can be expressed, but maybe that's a good lesson for students?

Reply:
Great questions. There's a tension with this API between enabling a high ceiling and the fact that the theater will be used relatively early in the course. I think what you're suggesting is pretty important for making the theater sufficiently capable in the realm of color manipulation, but I'm not clear on exactly how colors will be exposed to students early on. I'll check with the writers to see what they think.

Reply:
After a discussion with Jamila, we decided that we're going to keep string-based colors for now and, depending on use cases and time, add APIs/objects that allow for more powerful color handling. Two ideas that came up were:
1. `Color` class that could be initialized with ints or a string
2. A converter method from ints to string and from string to ints, so either can be used in pixel operations and drawing operations.

Reply:
that sounds great -- i agree with that approach completely 👍🏻

Reply:
For the future, I think it would be useful to have constants defined somewhere for common colors (similar to [java.awt.Color](https://docs.oracle.com/en/java/javase/11/docs/api/java.desktop/java/awt/Color.html).  Of the two ideas above, I think I like 1 but directing students to use constants (e.g. Color.BLACK) rather than construct a new one using a string.

Reply:
I discussed some of this with Jamila and we think there might be a way to use a `Color` class that makes the transition from string usage in Neighborhood easier using constants as you describe. I'll push an update with the proposal, stay tuned.